controller/types.go

```diff
@@ -20,77 +20,75 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
-const (
-	// StatusPassed define the pass value
-	StatusPassed = "Pass"
-	// StatusFailed define the Fail value
-	StatusFailed = "Fail"
-)
-
 var registeredResultMetrics []string
 
 // Declare the fixed chaos metrics. Dynamic (testStatus) metrics are defined in metrics()
 var (
-	ExperimentsTotal = prometheus.NewGaugeVec(prometheus.GaugeOpts{
-		Namespace: "c",
+	EngineTotalExperiments = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: "chaosEngine",
 		Subsystem: "engine",
-		Name:      "experiment_count",
+		Name:      "engine_experiment_count",
 		Help:      "Total number of experiments executed by the chaos engine",
 	},
-		[]string{"app_uid", "engine_name", "kubernetes_version", "openebs_version"},
+		[]string{"engine_namespace", "engine_name"},
 	)
 
-	PassedExperiments = prometheus.NewGaugeVec(prometheus.GaugeOpts{
-		Namespace: "c",
+	EnginePassedExperiments = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: "chaosEngine",
 		Subsystem: "engine",
-		Name:      "passed_experiments",
-		Help:      "Total number of passed experiments",
+		Name:      "engine_passed_experiments",
+		Help:      "Total number of passed experiments by the chaos engine",
 	},
-		[]string{"app_uid", "engine_name", "kubernetes_version", "openebs_version"},
+		[]string{"engine_namespace", "engine_name"},
 	)
 
-	FailedExperiments = prometheus.NewGaugeVec(prometheus.GaugeOpts{
-		Namespace: "c",
+	EngineFailedExperiments = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: "chaosEngine",
 		Subsystem: "engine",
-		Name:      "failed_experiments",
-		Help:      "Total number of failed experiments",
+		Name:      "engine_failed_experiments",
+		Help:      "Total number of failed experiments by the chaos engine",
 	},
-		[]string{"app_uid", "engine_name", "kubernetes_version", "openebs_version"},
+		[]string{"engine_namespace", "engine_name"},
 	)
-)
 
-// ExporterSpec contains the exporter related specs
-type ExporterSpec struct {
-	ChaosEngine      string
-	AppUUID          string
-	AppNS            string
-	OpenebsNamespace string
-}
+	ClusterTotalExperiments = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: "cluster",
+		Subsystem: "overall",
+		Name:      "cluster_experiment_count",
+		Help:      "Total number of experiments executed in the Cluster",
+	},
+		[]string{},
```

Comment:
should we add the cluster_uid here as the label? To filter out on per cluster basis from prom_ql queries.